Updated screenshots and descriptions to fit 2.2.x Deere skin (#154)
mixxxdj/manual: source/chapters/user_interface.rst

the main way to use Mixxx. In this chapter, we present the default interface of
Mixxx and describe its elements, knobs and faders.

> .. figure:: ../_static/2.2/overview.png

Author: Sure. Done.

> .. figure:: ../_static/2.2/overview.png

Reviewer: to avoid any possible trouble please make sure the screenshot only shows music you obtained legally ;)

there's still some stuff at the bottom.

> .. figure:: ../_static/2.2/overview.png

Author: Okay😅. The new screenshot shows a more legit music library.. Kindly crosscheck and see if its okay.

> .. figure:: ../_static/Mixxx-200-Deere.png

Reviewer 2: let's use lowercase file names. Also, the filename indicates the screenshot is from mixxx 2.0.0, but it's not. I'd prefer a naming scheme like this:

```suggestion
.. figure:: ../_static/2.2/interface.png
```

Author: True. I agree. The names were a bit too long.

   :align: center
   :width: 100%
   :figwidth: 100%
   :alt: The Mixxx interface - Deere skin
   :figclass: pretty-figures

   The Mixxx interface - Deere skin

> This is the Deere skin. It is the default skin supplied with Mixxx. There

Reviewer: ```suggestion
This is the Deere skin. It is the default skin supplied with Mixxx 2.2. There
```

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/interface-decks.png

Author: Done.

> .. figure:: ../_static/Mixxx-200-Deere-The-Deck-section.png

Reviewer 2: And here the name would be something like:

```suggestion
.. figure:: ../_static/2.2/interface-decks.png
```

Track Information Display
-------------------------

> .. figure:: ../_static/2.2/track-information.png

Author: done

            to the speed you are tapping, go to :ref:`control-keyboard` for more
            information.

**Cover Art**
  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the Mixxx library. 
  In Deere skin you can choose to enable or disable the cover art by clicking the gear icon in the upper right corner, then toggling the :menuselection:`Decks --> Cover Art` option. Clicking on the cover art will expand it into a full blown image.

**Key**
  The key displays the current musical key of the loaded track after pitch shifting. The key is displayed below the BPM (Tempo). This is the same as the key that is shown under the :guilabel:`Key` column heading in the Mixxx library. If it is hidden, you can drag the horizontal scroll bar at the bottom of the Mixxx library section, to the right, and the column will show. 
  Clicking the key will expand it to reveal more options: 

>   .. figure:: ../_static/2.2/key_options.png

Reviewer: The Match button is cropped. While this is a bug when Mixxx is at minimal size, please make the Mixxx window wider and take another screenshot.

Author: Okay, On it!

Author: Hi  @ronso0 here is a full screenshot from which i cropped the match button. My Mixxx window is maximized and taking up the entire screen but the Match button still displays as cropped.

![image](https://user-images.githubusercontent.com/28389950/96809280-3c1ee280-1423-11eb-9271-280a9209c7c2.png)

I also tried toggling off the some of the options in the Mixer settings to create more space so that it could display fully but it still didn't work. Maybe it's a Windows OS problem...? Is there another way i could get it to display fully?

Author: Alright then.

            to the speed you are tapping, go to :ref:`control-keyboard` for more
            information.

**Key**

>   .. figure:: ../_static/2.2/key-options.png

Reviewer: you can use this picture (intermediate fix sowe don't need to change the picture late ron)

![Screenshot_2020-10-29_18-46-41](https://user-images.githubusercontent.com/5934199/97612048-34f56700-1a17-11eb-966d-aa456d76cd8d.png)

Author: Okay.

  Clicking the key will expand it to reveal more options: 

>   .. figure:: ../_static/2.2/key_options.png

Reviewer 2: And please use a dash instead of an underscore in the filename (like in all other files).

>   .. figure:: ../_static/2.2/key_options.png

Author: okay... I've corrected this.

Clicking the key will expand it to reveal more options:

  * **Pitch Control**: This changes the track pitch independent of the tempo. Left-clicking the :guilabel:`-` button decreases the pitch by one semitone. Right-clicking it decreases the pitch by 10 cents. Left-clicking the :guilabel:`+` button increases the pitch by one semitone. Right-clicking it increases the pitch by 10 cents.
  * **Sync and Reset Key**: Left-clicking the :guilabel:`MATCH` button sets the key to a pitch that allows harmonic transition from the other track. It requires a detected key on both involved decks. Right-clicking it resets the key to the original track key.

**Cover Art**
  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the Mixxx library.

>   In Deere skin you can choose to enable or disable the cover art by clicking the gear icon in the upper right corner, then toggling the :menuselection:`Decks --> Cover Art` option. Clicking on the cover art will expand it into a full blown image.

Reviewer: ```suggestion
  In Deere skin you can choose to enable or disable the cover art by clicking the gear icon in the upper right corner, then toggling the :menuselection:`Decks --> Cover Art` option. Clicking on the cover art will expand it into a full blown image.
  The cover art is also shown in the Vinyl Widget next to the :ref:`parallel waveforms <interface-waveform>`.
```

            to the speed you are tapping, go to :ref:`control-keyboard` for more
            information.

**Key**
  .. figure:: ../_static/2.2/key-options.png
    :align: center
    :width: 42%
    :figwidth: 90%
    :alt: The options displayed when you click on the Key
    :figclass: pretty-figures

    The Key options.

> The key displays the current musical key of the loaded track after pitch shifting. The key is displayed below the BPM (Tempo). This is the same as the key that is shown under the :guilabel:`Key` column heading in the Mixxx library. If it is hidden, you can drag the horizontal scroll bar at the bottom of the Mixxx library section, to the right, and the column will show.

Author: Done

Clicking the key will expand it to reveal more options:

  * **Pitch Control**: This changes the track pitch independent of the tempo. Left-clicking the :guilabel:`-` button decreases the pitch by one semitone. Right-clicking it decreases the pitch by 10 cents. Left-clicking the :guilabel:`+` button increases the pitch by one semitone. Right-clicking it increases the pitch by 10 cents.
  * **Sync and Reset Key**: Left-clicking the :guilabel:`MATCH` button sets the key to a pitch that allows harmonic transition from the other track. It requires a detected key on both involved decks. Right-clicking it resets the key to the original track key.

**Cover Art**

>   The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the Mixxx library.

Reviewer: ```suggestion
  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the :ref:`Mixxx library <library-tracks>`.
```

Author: done.

Reviewer: Did you notice you can just click on the 'Commit suggestion' button and let the CI do the chcecking for you and double-check in the preview?
That's why I made those suggestions. No need to apply everything by hand locally...

Author: > > > Looking good, just some polishing left.
> > > Please also take care of @Holzhaus 's last review, then we can finally merge IMO.
> > 
> > 
> > Here is what i have in my code for those lines. My width is already at 25% but `figwidth` is at 60%. The new image that i placed here is much smaller.
> > Just to clarify, @Holzhaus do you need me to change the 25% to px?
> > ![image](https://user-images.githubusercontent.com/28389950/97743532-de5f5a00-1af6-11eb-839b-72902ba9dd19.png)
> 
> The question is: Why use a percentage if we use pixel widths for all other raster images? Please either use the pixel width of the image, or check what happens if you remove the `:width:` parameter. I suspect it's unnecessary to specify it at all, because I guess the image will just be displayed fullsize if missing.

Okay. I will try both and commit the one that works best.

Author: It turns out the width had no effect on how the image displayed. I have removed it.

Author: > Did you notice you can just click on the 'Commit suggestion' button and let the CI do the chcecking for you and double-check in the preview?
> That's why I made those suggestions. No need to apply everything by hand locally...

Oh. Wow. That would definitely make things much easier.  I don't know why i thought i didn't have permission to do this. I will use this button a lot more. Thanks!

Reviewer: Yeah, nice feature.
Yo can either commit every single suggestions or create a batch of them.

order to make :term:`beatmatching` easier, as it is possible to beatmatch
visually by aligning the beats that appear in each waveform.

.. figure:: ../_static/2.2/deere-parallel-waveforms.png
   :align: center
   :width: 100%
   :figwidth: 100%
   :alt: Mixxx default skin (Deere) - Parallel waveforms
   :figclass: pretty-figures

   Mixxx default skin (Deere) - Parallel waveforms

Depending on the skin, Mixxx displays either parallel waveforms (default) or
separate waveforms. Select your preferred appearance in
:menuselection:`Preferences --> Interface --> Skin`.

> .. figure:: ../_static/2.2/latenight-parallel-waveforms.png 

Reviewer 2: There already is a screenshot of Deere with parallel waveforms. Instead of a LateNight screenshot with parallel waveforms, this should be a screenshot of Deere with separate waveforms.

Author: Okay. Let me change this.

Author: I have made the requested changes @Holzhaus . Ready for your review... :)

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/options-button-grid.png

Author: done.

Playback/Cue Controls
---------------------

> .. figure:: ../_static/2.2/cue-controls.png

Author: done.

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

.. figure:: ../_static/2.2/transport-controls.png
  :align: center
  :width: 80px
  :figwidth: 100%
  :alt: The transport controls of the deck
  :figclass: pretty-figures

  The transport controls of the deck

**Beatjump/ Loop Move Size**

>   You can type or use the arrow keys in the input box to select the number of beats to jump or move, with the Beatjump Forward/ Backward buttons.

Reviewer: ```suggestion
  You can type, use the arrow keys or scroll vertically in the input box to select the number of beats to jump or move, with the Beatjump Forward/ Backward buttons.
```

**Beatjump Forward button**

>   Left-clicking this button while the loop is inactive will make the track jump forward by the set number of beats.
>   Left-clicking this button while the loop is active will move the loop forward by the set number of beats.
>   Right-clicking this button while the loop is inactive will make the track jump forward by one beat.
>   Right-clicking this button while the loop is active will move the loop forward by one beat.

Reviewer: I know we split it up like this in the tooltips, but for the text I find it easier to group by loop active/inactive.
So your addition is correct, though I think it's easier to grasp when written like this:
```suggestion
  Clicking this while the loop is inactive will make the track jump forward by the set number of beats (left click) or by one beat (right click).
  Clicking this while the loop is active will move the loop forward by the set number of beats (left click) or by one beat (right click).
```
Agree?

Reviewer: Also most of our users live with a left-to-right language, so we should mention the *Beatjump Backward button* first?

Author: Agreed. This format is easier on the eyes.  Less text and straight to the point.

**Beatjump Backward button**

>   Left-clicking this button while the loop is inactive will make the track jump backward by the set number of beats.
>   Left-clicking this button while the loop is active will move the loop backward by the set number of beats.
>   Right-clicking this button while the loop is inactive will make the track jump backward by one beat.
>   Right-clicking this button while the loop is active will move the loop backward by one beat.

Reviewer: ```suggestion
  Clicking this while the loop is inactive will make the track jump backward by the set number of beats (left click) or by one beat (right click).
  Clicking this while the loop is active will move the loop backward by the set number of beats (left click) or by one beat (right click).
```

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/transport-controls.png

Author: done.

 expletives on the fly, or simply as part of your mix.

>  .. note:: Not all of these controls may be present in all skins anymore (like the Deere skin). However, they can be mapped to keyboards and MIDI controllers. See this `guide here. <https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols>`_

Reviewer 2: ```suggestion
 .. note:: Not all of these controls may be present in all skins anymore (like the Deere skin). However, they can still be mapped to keyboards and MIDI controllers.

Reviewer 2: We'll add back the link as in internal reference in the 2.3 version of the manual.

Author: Okay.

.. sectionauthor::
   S.Brandt <s.brandt@mixxx.org>

.. figure:: ../_static/2.2/vinyl-control-mode.png
   :align: center

>    :width: 25%

Reviewer 2: Shouldn't this be a pixel width? By the way, maybe we don't need to specify the width at all?

Reviewer: @holzhaus Is this okay now? At least it looks good, dunno when this could be relevant
https://deploy-preview-154--mixxx-manual.netlify.app/chapters/user_interface.html#vinyl-control-mode-and-cueing-controls

Reviewer: _ping_ @Holzhaus

.. sectionauthor::
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/vinyl-control-mode.png

Author: Done.

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/sync-and-rate-controls.png

Author: done.

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/loop-controls.png

Author: done.

In this section of the Mixxx interface you can control (beat-)loops and set the

> loop points of a track.

Reviewer: ```suggestion
loop points of a track.
You can also shift loops using the :ref:`beatjump / loop move controls <transport-controls>` shown at the right side of the looping controls.
```

Author: Adding this change gives me this warning when i run `sphinx-build source build/html -b html -d build/doctrees -c source`

![image](https://user-images.githubusercontent.com/28389950/97626677-3e90c600-1a3b-11eb-8778-e66a6bfaa928.png)

Reviewer: oh my bad, wanted to help but pasted the wrong string.
it needs to ref to Transport Controls. the ref is line 394 `interface-transport`.
I adapted the code suggestion accordingly.

**Beatlooping Buttons**

* **Instant loop**: The input box allows you to set any BeatLoop Size, each of which represents a different :term:`bar`

>   length. This value will set a loop of the defined

Author: Done.

  or not. This works for manually placed loops as well as automatic loops set by
  the beatlooping buttons. Depending on the current status of the loop, the
  loop overlay on the waveforms changes color.
* **ReLoop**:
  Temporarily sets up a rolling loop over the defined number of beats. 

>   Playback will resume where the track  would have been if it had not entered the loop.

Reviewer: in the entire text above, could you please remove the headers Beatlooping Buttons and Loop Buttons headers and format Instant Loop, ReLoop & Loop-In/Out like the controls in the rate chapter?
![image](https://user-images.githubusercontent.com/5934199/97328669-89151580-1876-11eb-9ad6-fce323cbbee3.png)

Can you also try to format the Sync button string like Pitch Slider?

>   Playback will resume where the track  would have been if it had not entered the loop.

Author: @ronso0  The tooltips call the Instant Loop a Beatloop. Won't this cause confusion for new Mixxx users? Maybe we can put it as Instant Loop/ Beatloop ? I looked for the Instant Loop button by hovering over the buttons and reading the tooltips and couldn't find it until much later when i realized it was the Beatloop (as written in the tooltip).

>   Playback will resume where the track  would have been if it had not entered the loop.

Reviewer: Okay, I didn't double-check. Please stick to the tooltips then.

>   Playback will resume where the track  would have been if it had not entered the loop.

Author: Alright

   RJ Ryan <rryan@mixxx.org>
   S.Brandt <s.brandt@mixxx.org>

> .. figure:: ../_static/2.2/cue-controls.png

Reviewer: this is the wrong picture, we need one of the hotcue grid here (with 8 hotcues)
![image](https://user-images.githubusercontent.com/5934199/97329042-eb6e1600-1876-11eb-9040-bae9ff9879cd.png)

Author: Okay. Corrected it

   .. figure:: ../_static/Mixxx-200-Latenight-Bottom-Expansion-Buttons.png
      :align: center
      :width: 448px
      :figwidth: 100%
      :alt: The section expansion buttons
      :figclass: pretty-figures
  .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
    :align: center
    :width: 245px
    :figwidth: 90%
    :alt: The section expansion buttons
    :figclass: pretty-figures

      The section expansion buttons on the bottom
    The section expansion options under the Skin settings on the right.

On top, and on the button of the user interface are the section expansion buttons.
On top, and on the right side of the user interface are the section expansion buttons and options.

If you click on either :guilabel:`MIC`, :guilabel:`SAMPLER`, or
:guilabel:`VINYL` then you will enable control sections for interacting with:
If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:

* :ref:`interface-mic`
* :ref:`interface-effects`
* :ref:`interface-sampler`
* :ref:`interface-vc-mode`
* :ref:`interface-mic`

You can also use the specific :ref:`appendix-shortcuts`.

> 

Reviewer 2: Please revert this change. The missing newline makes the anchor below invalid, which is why the build error occurs:

```
/opt/build/repo/source/chapters/example_setups.rst:30:undefined label: interface-head-master (if the link has no caption the label must precede a section header)
```

Author: Oh. Thanks! I had missed that.

Section Expansion Buttons
-------------------------

.. figure:: ../_static/2.2/expansion-buttons-top.png
   :align: center
   :width: 435px
   :figwidth: 100%
   :alt: The section expansion buttons
   :figclass: pretty-figures

   The section expansion buttons on top

   .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
      :align: center
      :width: 220px
      :figwidth: 100%
      :alt: The section expansion buttons
      :figclass: pretty-figures

      The section expansion options under the Skin settings on the right.

On top, and on the right side of the user interface are the section expansion buttons abd options.

> If you click on either :guilabel:`MICS & AUX`, :guilabel:`SAMPLER`, or
> :guilabel:`EFFECTS` then you will enable control sections for interacting with:

Reviewer: let's describe in order of appearance:
```suggestion
If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLER` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
```

Author: Okay.

* :ref:`interface-mic`
* :ref:`interface-sampler`

> 

Reviewer: nice catch!

Author: Thanks! :)

> 

Reviewer: ```suggestion
:ref:`interface-effects`
```
